app/frontend/wallet/shelley/shelley-transaction.ts

```diff
@@ -51,13 +81,18 @@ function ShelleyTxAux(
     txBody.set(TxBodyKey.INPUTS, cborizeTxInputs(inputs))
     txBody.set(TxBodyKey.OUTPUTS, cborizeTxOutputs(outputs))
     txBody.set(TxBodyKey.FEE, fee)
-    txBody.set(TxBodyKey.TTL, ttl)
+    if (ttl || ttl === 0) {
```

Comment:
I think this is equivalent to `ttl != null`

Reply:
Fixed.